examples/integration/test/fixtures/bwb.xcodeproj/project.pbxproj

```diff
@@ -21450,10 +21450,8 @@
 				COMPILE_TARGET_NAME = iOSApp.library;
 				DEBUG_INFORMATION_FORMAT = "dwarf-with-dsym";
 				DEPLOYMENT_LOCATION = NO;
-				DEVELOPMENT_TEAM = "$(inherited)";
 				"DEVELOPMENT_TEAM[sdk=iphoneos*]" = V82V4GQZXM;
-				EXECUTABLE_NAME = iOSApp_ExecutableName;
-				"EXECUTABLE_NAME[sdk=iphoneos*]" = "$(inherited)";
+				"EXECUTABLE_NAME[sdk=iphonesimulator*]" = iOSApp_ExecutableName;
```

Comment:
Before this change, the `$(inherited)` for the device build would use the same value as the simulator build.